everviz/pages/summary_values.py

```diff
@@ -65,34 +63,38 @@ def _set_up_data_sources(api, keys=None):
 
     logger.info("Generating summary values plot")
     summary_values_file = os.path.join(everviz_path, "summary_values.csv")
-    data = _summary_values(summary_values)
-    data.to_csv(summary_values_file, index=False, sep=";")
+    summary_statistics_file = os.path.join(everviz_path, "summary_statistics.csv")
+    values, statistics = _summary_values(summary_values)
+    values.to_csv(summary_values_file, index=False)
+    statistics.to_csv(summary_statistics_file, index=False)
     logger.info(f"File created: {summary_values_file}")
+    logger.info(f"File created: {summary_statistics_file}")
 
-    return DataSources(summary_values=summary_values_file,)
+    return DataSources(
+        summary_values=summary_values_file, summary_statistics=summary_statistics_file,
+    )
 
 
 def page_layout(api):
     sources = _set_up_data_sources(api)
-    if sources is None:
```

Comment:
Can sources no longer be None?

Reply:
That would be the case that an optimization does not use eclipse or flow and does not produce summary values.

Reply:
So we need that check, running the plugin on a case without summary values gives `AttributeError`.

Can you also add a test simulating that condition?

Reply:
done.